src/nuid/datalog/portal.cljc

```diff
@@ -67,23 +68,16 @@
    {:db/ident       :nuid.portal.notification/addresses
     :db/valueType   :db.type/ref
     :db/cardinality :db.cardinality/many
-    :db/doc         "List of notification addresses"}])
+    :db/doc         "List of notification addresses"}
 
-#_(def org
-    [{:db/ident       :nuid.portal/developers
-      :db/valueType   :db.type/ref
-      :db/cardinality :db.cardinality/many
-      :db/isComponent true
-      :db/doc         "Set of [[:nuid.portal/developer]]s"}
-
-     {:db/ident       :nuid.portal.org/administrators
-      :db/valueType   :db.type/ref
-      :db/cardinality :db.cardinality/many
-      :db/isComponent true
-      :db/doc         "Administrators of a [[:nuid.portal/org]]"}])
+   {:db/ident       :nuid.portal/subscriptions
+    :db/valueType   :db.type/ref
+    :db/cardinality :db.cardinality/many
+    :db/doc         "Set of [[:nuid.portal/subscription]]s"}])
```

Comment:
@localshred moved this to [roam](https://roamresearch.com/#/app/nuid/page/WuX2qQiAB) as a sketch to reduce noise in this file. that caused the diff to look weird, but the thing to note is that one new attribute `:nuid.portal/subscriptions` was added to `nuid.datalog.portal/attributes`.